src/eva/tests/config/testCubedSphereRestart.yaml

```diff
@@ -0,0 +1,66 @@
+diagnostics:
+
+  # Data read
+  # ---------
+- data:
+    type: CubedSphereRestart
+    datasets:
+      - name: experiment
+        variable: T
+        resolution: C768
+        fv3_filenames:
+          - ${data_input_path}/20210801.000000.sfc_data.tile1.nc
+          - ${data_input_path}/20210801.000000.sfc_data.tile2.nc
+          - ${data_input_path}/20210801.000000.sfc_data.tile3.nc
+          - ${data_input_path}/20210801.000000.sfc_data.tile4.nc
+          - ${data_input_path}/20210801.000000.sfc_data.tile5.nc
+          - ${data_input_path}/20210801.000000.sfc_data.tile6.nc
+        orog_filenames:
+          - ${data_input_path}/C768_oro_data.tile1.nc
+          - ${data_input_path}/C768_oro_data.tile2.nc
+          - ${data_input_path}/C768_oro_data.tile3.nc
+          - ${data_input_path}/C768_oro_data.tile4.nc
+          - ${data_input_path}/C768_oro_data.tile5.nc
+          - ${data_input_path}/C768_oro_data.tile6.nc
+        groups:
+          - name: FV3NcDiag
```

Comment:
NcDiag is not an appropriate group name here I don't think, go with `FV3Restart` maybe?